src/nile/core/node.py

```diff
@@ -1,25 +1,20 @@
 """Command to start StarkNet local network."""
-import json
 import logging
 import subprocess
 
-from nile.common import NODE_FILENAME, DEFAULT_GATEWAYS
+from nile.common import create_node_json
 
 
 def node(host="127.0.0.1", port=5050, seed=None, lite_mode=False):
```

Comment:
> Having two local networks configured shouldn't be allowed?

IMO it's not necessary. Executing `nile node` while a local node is already running would indeed overwrite the dictionary. We can add a check to prevent that. If we want to allow multiple local nodes, we'll need to give a deeper refactor on network handling

Reply:
I'm not worried about overwriting the localhost config, but this is overwriting the whole file.

What if I had **mainnet-fork** network configured? (this would be possible soon), but I will lose it by just running `nile node`.

I think a `nile.ini` file is a stronger solution for this.